nncf/tensorflow/README.md

```diff
@@ -29,20 +29,28 @@ The samples demonstrate the usage of compression algorithms for two different us
 
 ## Usage
 The NNCF TF is organized as a regular Python package that can be imported in an arbitrary training script.
-The basic workflow is loading a JSON configuration script containing NNCF-specific parameters determining the compression to be applied to your model, and then passing your model along with the configuration script to the `nncf.create_compressed_model` function.
+The basic workflow is loading a JSON configuration script containing NNCF-specific parameters determining the compression to be applied to your model, and then passing your model along with the configuration script to the `nncf.tensorflow.create_compressed_model` function.
 This function returns a transformed model ready for compression fine-tuning, and handle to the object allowing you to control the compression during the training process:
 
 ```python
-import nncf
-from nncf.tensorflow import create_compressed_model
+import tensorflow as tf
+
 from nncf import NNCFConfig
+from nncf.tensorflow import create_compressed_model
+from nncf.tensorflow.initialization import register_default_init_args
```

Comment:
Maybe add 
```python
from nncf.tensorflow.initialization import register_default_init_args
```
to `tensorflow/__init__.py` for consistency with `torch`

Change this line to 
```python
from nncf.tensorflow import register_default_init_args
```

Reply:
Thank you for your comment. If you look at the PyTorch examples you will see that the register_default_init_args helper function is imported the following way 'from nncf.torch.initialization import register_default_init_args', for example
https://github.com/openvinotoolkit/nncf/blob/76e4a0c0f6a63de77d601e8da2bd615f99101242/examples/torch/classification/main.py#L54

https://github.com/openvinotoolkit/nncf/blob/76e4a0c0f6a63de77d601e8da2bd615f99101242/examples/torch/object_detection/main.py#L35

https://github.com/openvinotoolkit/nncf/blob/76e4a0c0f6a63de77d601e8da2bd615f99101242/examples/torch/semantic_segmentation/main.py#L40

I can guess that this usage is more easily. Thus I aligned the readme with the real examples.

Reply:
Ok. I think in the future we need  to align `torch` and `tensorflow`. Remove 
```python
from nncf.tensorflow.initialization import register_default_init_args
```
from `torch/__init__.py`.

It seems strange that we can 
```python
from nncf.torch import register_default_init_args
```
but can't
```python
from nncf.tensorflow import register_default_init_args
```

Reply:
I will add this possibility, but will not update examples.